src/governance.ts

```diff
@@ -128,8 +134,54 @@ export class GovernanceCanister {
     return this.responseConverters.toListProposalsResponse(rawResponse);
   };
 
+  public createNeuron = async ({
+    stake,
+    principal,
+    ledgerCanister,
+  }: {
+    stake: ICP;
+    principal: Principal;
+    ledgerCanister: LedgerCanister;
+  }): Promise<void> => {
+    if (stake.toE8s() < E8S_PER_ICP) {
+      throw "Need a minimum of 1 ICP to stake a neuron";
+    }
+
+    const nonceBytes = new Uint8Array(randomBytes(8));
+    uint8ArrayToBigInt;
+    const nonce = uint8ArrayToBigInt(nonceBytes);
+    const toSubAccount = buildSubAccount(nonceBytes, principal);
+
+    // Note - the account identifier here is just an alias of string.  Something to change?
+    const accountIdentifierAsString = principalToAccountIdentifier(
+      this.canisterId,
+      toSubAccount
+    );
+    const accountIdentifier = AccountIdentifier.fromHex(
+      accountIdentifierAsString
+    );
+
+    // Send amount to the ledger.
+    await ledgerCanister.transfer({
```

Comment:
This method returns errors, which I think should be handled. I am also OK with doing that in a later PR, but if so we should definitely track it to make sure it isn't forgotten.

Reply:
ok, I will wrap all the calls with a try/catch block and add a comment to manage properly once we agree on the pattern.